From d280cc510d379d6b06a405895ac35a2af7b1aa58 Mon Sep 17 00:00:00 2001
From: Dmytro Kelzikov <d.kelzikov@inango-systems.com>
Date: Tue, 2 Mar 2021 23:38:33 +0200
Subject: [PATCH] Commit Title: Fix compilation errors in files openvswitch
 package

BugID: 23185
Bug Fixed(y/N[blankN]): N
CodeReviewId: 0
Description: Fixed compilation errors in ip6_gre.c and ip_tunnel.c files.
---
 datapath/linux/compat/ip6_gre.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/datapath/linux/compat/ip6_gre.c b/datapath/linux/compat/ip6_gre.c
index bbb9731e6..7fd345309 100644
--- a/datapath/linux/compat/ip6_gre.c
+++ b/datapath/linux/compat/ip6_gre.c
@@ -989,7 +989,7 @@ static netdev_tx_t ip6erspan_tunnel_xmit(struct sk_buff *skb,
 					 struct net_device *dev)
 {
 	struct ip6_tnl *t = netdev_priv(dev);
-//	struct dst_entry *dst = skb_dst(skb);
+	struct dst_entry *dst = skb_dst(skb);
 	struct ip_tunnel_info *tun_info;
 	const struct ip_tunnel_key *key;
 	struct net_device_stats *stats;
@@ -1086,7 +1086,7 @@ static netdev_tx_t ip6erspan_tunnel_xmit(struct sk_buff *skb,
 
 	/* Push GRE header. */
 	gre_build_header(skb, &tpi, 8);
-// fixed compiler error: too few arguments to function ‘dst->ops->update_pmtu’, don't relate to brcompate.
+
 	/* TooBig packet may have updated dst->dev's mtu */
 	if (!t->parms.collect_md && dst && dst_mtu(dst) > dst->dev->mtu)
 #ifndef HAVE_DST_OPS_CONFIRM_NEIGH
